### DISCUSSION TITLE
Add option to disable "Unchanged files with check annotations"

### DISCUSSION THREAD

**Discussion by berniedurfee-renaissance** [community/community#81472] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

I appreciate the intent with this feature, but it really should be optional, as it's distracting for reviewers.

**Comment by fdwr** [comment]
Indeed, this causes a lot of irrelevant noise in code reviews (in files in 5 year old files that I have no authority to change anyway).

**Comment by xezon** [comment]
Yes please. This feature needs to be opt-in or opt-out.
